gobblin-core/src/main/resources/mysql/mysql.properties

```diff
@@ -16,7 +16,7 @@
 #
 
 # Source properties - source class to extract data from Mysql Source
-source.class=gobblin.source.extractor.extract.jdbc.MysqlSource
+source.class=gobblin.source.jdbc.MysqlSource
```

Comment:
With calcite introduced, all deployed application will need to be updated, adding a new dependency. 

To me, the `MysqlSource` was given a bad fully qualified name, so are some other sql related components. My plan is to piggyback those name changes on adding the calcite as a new dependency while updating the application.